scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -627,15 +660,15 @@ Importantly, one reason that batches may still appear separated in the corrected
 Recall from earlier that we conveniently have cell type annotations in our SCEs, so we can explore those here!
 Let's take a quick detour to see what kinds of cell types are in this data by making a barplot of the cell types across samples:
 
-```{r explore celltypes}
-# Cell types are stored in the `celltype_broad` and `celltype_fine` columns in the SCE
+```{r explore_celltypes, live = TRUE}
```

Comment:
I might not do this live. I don't think we want people to have to reload ggplot syntax and options at this moment in the course.